public/js/lib/editor/index.js

```diff
@@ -121,30 +122,6 @@ export default class Editor {
       },
       'Shift-Ctrl-Backspace': cm => {
         utils.wrapTextWith(this.editor, cm, 'Backspace')
-      },
```

Comment:
I think I'd like to preserve this, since it allows more selective application of wrapping.

Reply:
The problem with `C-\` was its destructive nature towards the lists/blockquotes, headings,etc in the selection. Except straight line paragraphs (which is an uncommon use case for codimd) it would always wrap incorrectly.

Reply:
Fair.